interop_tests/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10.7
+
+# Install interop-tests repo
+RUN pip3 install git+https://github.com/rhpit/interop-tests.git@main
+
+# Upgrade pip and install required packages
+RUN pip install --upgrade pip
+RUN pip install openshift-python-wrapper
```

Comment:
```suggestion
```

These can be removed @ascerra now that `openshift-python-wrapper` is installed when this package is installed above on line 4.